account_product_fiscal_classification/tests/tests.py

```diff
@@ -138,3 +144,12 @@ def test_06_unlink_fiscal_classification(self):
             [self.classification_1_id])
         with self.assertRaises(ValidationError):
             classification.unlink()
+
+    def test_07_classification_generate_from_template(self):
```

Comment:
@pedrobaeza it happens that I'm currently working on a very poor hardware these days and looking at Travis directly is easier for me, well at least was until I understood it wasn't trivial. Trying a rename now..